validator/client/validator.go

```diff
@@ -34,7 +34,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
-type validatorRole int8
+// ValidatorRole defines the validator role.
+type ValidatorRole int8
```

Comment:
Had to expose this or linter complains:
`
validator/client/mock_validator.go:122:67: exported method RolesAt returns unexported type map[[48]byte][]client.validatorRole, which can be annoying to use`